Tilers/ObjTiler/obj.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+import sys
+import numpy as np
+import pywavefront
+from py3dtiles import BoundingVolumeBox, TriangleSoup
+from Tilers.object_to_tile import ObjectToTile, ObjectsToTile
+
+import os
+from os import listdir
+from os.path import isfile, join
+
+
+
+class Obj(ObjectToTile):
```

Comment:
Maybe place some py-doc string explaining that Obj refers to
the OBJ file format would be of some help ? (otherwise one might
imagine that obj might be say some alternative replacement for the 
python "object" class)

Reply:
Done in [#4e7ecd4](https://github.com/VCityTeam/py3dtiles/pull/5/commits/4e7ecd4bef4e9dfa054808e06e3bbc263cc69b3d)